plugins/aws-s3-storage/src/s3PackageManager.ts

```diff
@@ -242,15 +245,20 @@ export default class S3PackageManager implements ILocalPackageManager {
       err => {
         if (err) {
           const convertedErr = convertS3Error(err);
-          this.logger.error({ convertedErr }, 's3: [S3PackageManager writeTarball headObject] @convertedErr');
+          this.logger.debug({ convertedErr }, 's3: [S3PackageManager writeTarball headObject] @{convertedErr}');
 
           if (is404Error(convertedErr) === false) {
-            this.logger.error({ convertedErr }, 's3: [S3PackageManager writeTarball headObject] non a 404 emit error');
+            this.logger.error(
+              { convertedErr },
+              's3: [S3PackageManager writeTarball headObject] @{convertedErr} non a 404 emit error'
+            );
 
             uploadStream.emit('error', convertedErr);
           } else {
             this.logger.debug('s3: [S3PackageManager writeTarball managedUpload] init stream');
-            const managedUpload = this.s3.upload(Object.assign({}, baseS3Params, { Body: uploadStream }));
+            const managedUpload = this.s3.upload(
+              Object.assign({}, baseS3Params, { Body: uploadStream, ACL: this.tarballACL })
+            );
```

Comment:
This is the core logic of this PR, passing tarballACL when uploading a tarball file.